almanach/common/exceptions/authentication_failure_exception.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
This is wierd when I look at the the diff it was not there but it is there in the unified view...